lib/BC/Util.cpp

```diff
@@ -1204,46 +1204,6 @@ MoveConstantIntoModule(llvm::Constant *c, llvm::Module *dest_module,
         moved_c = ret;
         return ret;
       }
-      case llvm::Instruction::UDiv: {
```

Comment:
These operations have been removed as constant expressions in LLVM 15. See the linked release notes for details.